Cannot file module .meteor/release/METEOR@0.9.1.release.json

I'm following this [this tutorial](http://mherman.org/blog/2014/01/29/meteor-dot-js-in-action-create-an-app-test-with-laika/#.VAmrrmSSzKg) and when I try to run Laika test I get the output bellow.

I'm new to meteor and Laika so I might configure something wrong.

```
  injecting laika...
  loading phantomjs...
  loading initial app pool...
  cleaning up injected code


module.js:340
    throw err;
          ^
Error: Cannot find module '/Users/me/.meteor/releases/METEOR@0.9.1.release.json'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:280:25)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at Object.getMeteorToolsPath (/usr/local/lib/node_modules/laika/lib/helpers.js:33:18)
    at Object.getMeteorNode (/usr/local/lib/node_modules/laika/lib/helpers.js:43:35)
    at Object.getMeteoriteNode (/usr/local/lib/node_modules/laika/lib/helpers.js:59:20)
    at detectConfig (/usr/local/lib/node_modules/laika/lib/app_pool.js:83:35)
    at new AppPool (/usr/local/lib/node_modules/laika/lib/app_pool.js:16:19)
    at /usr/local/lib/node_modules/laika/bin/_laika:90:17
```

Looks like a change in Meteor >= 0.9.0 doesn't generate the expected file anymore for current Meteor version. I am on Meteor 0.9.2 and get `Cannot find module '/Users/xxx/.meteor/releases/METEOR@0.9.2.1.release.json'`.

I have the following files:

``` bash
> ls ~/.meteor/releases/
0.8.3.notices.json
0.8.3.release.json
0.9.1.1.release.json
latest -> 0.9.1.1.release.json
```

@apendua do you mean the XCode Command Line Tools? These are already installed.  
Just tried on an freshly created meteor project and still got   

```
Cannot find module '/Users/dfucci/.meteor/releases/METEOR@0.9.3.release.json'
```

--- a/lib/helpers.js
+++ b/lib/helpers.js
@@ -30,10 +30,27 @@ exports.getRandomPort = function getRandomPort() {
 
 exports.getMeteorToolsPath = function getMeteorToolsPath() {
   var version = fs.readFileSync('.meteor/release').toString().trim();
+
+  // support both the old and the new (METER@x.x.x) version style
+  version = version.split('@')[1] || version;
+
   var releases = require(util.format('%s/.meteor/releases/%s.release.json', process.env.HOME, version));
 
-  var toolsPath = util.format('%s/.meteor/tools/%s', process.env.HOME, releases.tools);
-  return toolsPath;
+  // even though this comparision method is really naive, it should work just fine
+  if (version < "0.9.0") {
+    return util.format('%s/.meteor/tools/%s', process.env.HOME, releases.tools);
+  }
+
+  // this is basically the same thing the original meteor script is doing
+  // (see ~/.meteor/meteor) to find out the correct path to node exectuable
+
+  var meteorHome = path.resolve(process.env.HOME, '.meteor');
+  var meteorLink = path.resolve(meteorHome, fs.readlinkSync(path.join(meteorHome, 'meteor')));
+
+  meteorLink = meteorLink.split(path.sep);
+  meteorLink[meteorLink.length - 1] = 'dev_bundle';
+
+  return path.sep + path.join.apply(path, meteorLink);
 }
 
 exports.getMeteorNode = function getMeteorNode() {

--- a/lib/helpers.js
+++ b/lib/helpers.js
@@ -34,11 +34,12 @@ exports.getMeteorToolsPath = function getMeteorToolsPath() {
   // support both the old and the new (METER@x.x.x) version style
   version = version.split('@')[1] || version;
 
-  var releases = require(util.format('%s/.meteor/releases/%s.release.json', process.env.HOME, version));
+  var release;
 
   // even though this comparision method is really naive, it should work just fine
   if (version < "0.9.0") {
-    return util.format('%s/.meteor/tools/%s', process.env.HOME, releases.tools);
+    release = require(util.format('%s/.meteor/releases/%s.release.json', process.env.HOME, version));
+    return util.format('%s/.meteor/tools/%s', process.env.HOME, release.tools);
   }
 
   // this is basically the same thing the original meteor script is doing

